src/pages/Cart/index.tsx

```diff
@@ -16,6 +16,7 @@ const Cart = () => {
   const isDisable = true;
   const [isCheckout, setIsCheckout] = useState(false);
   const navigate = useNavigate();
+  const [showOverlay, setShowOverlay] = useState(false); // State for overlay
```

Comment:
Remove redundant comment, your naming is good enough

Reply:
Fixed